### DISCUSSION TITLE
Alias Contact Manager Display Name

### DISCUSSION THREAD

**Discussion by asentris** [simple-login/app#1427] [Feature Requests]
Is it possible to view the display name of a reverse-alias in the contact list of an alias? Right now I can only see the emails, but many times the email doesn't indicate the name of a personal contact, so it can be difficult to find the email I need without searching.

**Comment by meilcowski** [comment]
I agree, rarely does a persons name actually show up in their email address, so If I search for the name in the reverse alias list it wouldnt come back. Even just a notes field would solve the problem as I need to be able to associate a name with an address for each reverse alias.
